docs/book/src/plugins/plugins.md

```diff
@@ -19,7 +19,7 @@ This section details how to extend Kubebuilder and create your plugins following
 <aside class="note">
 <h1>Note</h1>
 
-For further information check the design proposal docs [Extensible CLI and Scaffolding Plugins: phase 1][plugins-phase1-design-doc] and [Extensible CLI and Scaffolding Plugins: phase 1.5][plugins-phase1-design-doc-1.5]. 
+ You can check the existing design proposal docs at [Extensible CLI and Scaffolding Plugins: phase 1][plugins-phase1-design-doc] and [phase 1.5][plugins-phase1-design-doc-1.5]. To know more about the future vision, check [Extensible CLI and Scaffolding Plugins: phase 2][plugins-phase2-design-doc]. 
```

Comment:
```suggestion
You can check the existing design proposal docs at [Extensible CLI and Scaffolding Plugins: phase 1][plugins-phase1-design-doc] and it's [phase 1.5][plugins-phase1-design-doc-1.5] to know more about what is provided by Kubebuilder CLI and API currently. 
```

That is not the only material in the future vision subsection. So, I think the best here is do a new note to just clarifies that.